app/views/foods/_food.html.erb

```diff
@@ -0,0 +1,10 @@
+<tr class="food-item">
+    <td><%= food.name%></td>
+    <td><%= food.measurement_unit%></td>
+    <td><%= food.price%></td>
+    <td>
+        <% if can? :destroy, Food %>
+            <%=  button_to 'Delete', delete_food_path(food.id), method: :delete %>
+        <% end %>
+    </td>
+</tr>
```

Comment:
- [ ] as part of the requirements kindly make sure to follow the given wireframe for the food list page 

Your Design         | After
:-------------------------:|:-------------------------:
![image](https://github.com/Tshobohwa/recipe-app/assets/89282221/4e913cce-6700-49e5-97c9-c8eb402b9918) | ![image](https://github.com/Tshobohwa/recipe-app/assets/89282221/32ba67c2-8876-4425-9421-8e05d45edf02)